### ISSUE TITLE
Slider is inaccessible in Windows High Contrast Mode

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#6859]
The slider component is not accessible with Windows High Contrast Mode enabled in either Firefox or Edge.

## Environment

> Operating system

Windows 10

> Browser

Firefox, Edge

> Assistive technology used to verify

Windows High Contrast Mode

## Detailed description

![image](https://user-images.githubusercontent.com/40970507/93649894-ab438a00-f9d2-11ea-8515-0fc445ad6b41.png)
